### DISCUSSION TITLE
Does this depend on the docker container to work?

### DISCUSSION THREAD

**Discussion by talormanda** [mrlt8/docker-wyze-bridge#601] [General]
Question about how this works...does all connections go through the docker container to function? Meaning I shouldn't host it on a weak device?

**Comment by mrlt8** [comment]
Audio and video streams from the camera need to go through the container. 

You don't necessarily need lots of processing power or ram If you're not doing any rotation to the video since the raw h264 frames will be copied into an rtsp stream. Audio _may_ need to get re-encoded depending on the camera firmware as some audio codecs are not compatible with the rtsp format.
